llama-index-integrations/llms/llama-index-llms-predibase/llama_index/llms/predibase/base.py

```diff
@@ -98,12 +115,23 @@ def __init__(
             output_parser=output_parser,
         )
 
+        self.adapter_id = adapter_id
```

Comment:
since adaptor ID is part of the pydantic model of the LLM class, its better practice to pass this into the `super().__init__(..., adapter_id=adapter_id)` line